ffi/src/log.rs

```diff
@@ -0,0 +1,33 @@
+static LOG_INITED: std::sync::atomic::AtomicBool = std::sync::atomic::AtomicBool::new(false);
+
+#[diplomat::bridge]
+pub mod ffi {
+    use std::{fs::File, path::PathBuf};
+
+    use super::LOG_INITED;
+    use std::fs;
+
+    #[diplomat::opaque]
+    pub struct Log;
+
+    impl Log {
+        pub fn init_log_with_env() {
```

Comment:
naming:
```suggestion
        pub fn init_with_env() {
```

Will give `Log.InitWithEnv()`